states/new_york/main.py

```diff
@@ -28,7 +28,7 @@ def parse_county(soup):
     results['faxes'] += re_phone.findall(fax_line)
 
   # emails
-  results['emails'] = list(set(i.get('href').replace('mailto:','').strip() for i in blocks[0].select('a[href^=mailto]')))
+  results['emails'] = sorted(set(i.get('href').replace('mailto:','').strip() for i in blocks[0].select('a[href^=mailto]')))
```

Comment:
Instead of handling each of these cases, consider writing a 'normalizer' function in common that introspects the types and performs these sorts and dedupes as the last step for each state.  You don't even need to do recursion since the objects are not deep.  You could then import and run this function for every state right before you save.  Each state's outputs are the same so deduping is easy.

Reply:
Good suggestion. I will work on a common normalize_state function tomorrow.

Reply:
BTW - if you've addressed everything, we can also save this suggestion for another diff.  It is a bit of scope creep.

Reply:
Will address in a separate diff / pr.